test/functional/index.test.js

```diff
@@ -1350,4 +1353,94 @@ describe('Indexes', function() {
       });
     }
   });
+
+  context('commitQuorum', function() {
+    function throwErrorTest(testCommand) {
+      return {
+        metadata: { requires: { mongodb: '<4.4' } },
+        test: function(done) {
+          const client = this.configuration.newClient();
+          client.connect(err => {
+            expect(err).to.not.exist;
+            const db = client.db('test');
+            const collection = db.collection('commitQuorum');
+            testCommand(db, collection, (err, result) => {
+              expect(err).to.exist;
```

Comment:
if any of these expectations fail then you will fail to close the client, this is why `withClient` was originally written, maybe it could be helpful here as well?